### DISCUSSION TITLE
Can a Webview bind ”whatever VS Code binds” to a command?

### DISCUSSION THREAD

**Discussion by PEZ** [microsoft/vscode-discussions#318] [Extension Development QnA] [answered]
Say I have a WebView that can go to definition of something. So if I had that something in a regular text editor, I would hit F12 to go to the definition, or some other keys if I have re-bound the go-to-definition command.

Can I make the WebView go to definition of the thing on F12 (or whatever I/the user has re-bound to)?

**Comment by burkeholland** [comment]
Is this still an open question? If you've resolved, can you update with the answer?

**Reply by PEZ** [reply]
I still wonder about this.

**Reply by isidorn** [reply, marked as answer]
This is currently not possible. Though a fair feature request for the Webview.
